apps/auth_zero/tests.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from apps.common.config import ConfigFileLoader
+from apps.common.config import CredentialsLoader
+
+
+class UserTestCase(TestCase):
+    def setUp(self):
+        self.user_model = get_user_model()
+        config_file = "apps/auth_zero/config/data/test/test_config.ini"
+        self.creds_obj = CredentialsLoader(config_file, ConfigFileLoader)
+        self.users = {
+            verification_status: self.user_model.objects.create_user(
```

Comment:
when you create a user/interact outside of application logic boundary, it is no longer a "unit" test; this is an integration test;
You could try using interfaces that mock or rather override functionality so that actual DB operations are mimicked and some stubs take over; we assume that they do certain things and we can define those implementations;

Reply:
Could you give an example, please?

Reply:
Is this still an issue?

> 
> Could you give an example of how we could implement such interfaces in relation to the tests shown above, please?

Reply:
> Is this still an issue?
> 
> > Could you give an example of how we could implement such interfaces in relation to the tests shown above, please?

Do I need to rewrite the tests, or are we proceeding with the currently written tests?

Reply:
> Do I need to rewrite the tests, or are we proceeding with the currently written tests?

If the program is not accessing anything outside its own boundary; if you can run a Django test in flight mode with no SQL/NoSQL connected, then its a unit test;

Reply:
Should I still work on this or will this not be applicable for the new repo?